doc/creating_a_release.md

```diff
@@ -0,0 +1,12 @@
+# Creating a release
+
+These releases represent an upstream release and not a release of the product.
+
+- Ensure you are on the master branch
+- Ensure git is not in a dirty state (ie stuff needs to be committed )
```

Comment:
Should we add that we should run integration tests first?

I am hoping, that the level of integration testing that would be suitable for comunity releases would happen in CI, but that is not the case yet.

Reply:
It should not have been merged to master if the tests were failing so I see little benefit in re running them again ping @witmicko